packages/backend/src/apps/toolbox/common/condition-is-true.ts

```diff
@@ -36,9 +49,7 @@ export default function conditionIsTrue(conditionArgs: IJSONObject): boolean {
       result = field === null || field === undefined || field === ''
       break
     default:
```

Comment:
nit: would recommend falling through instead of using a default block, it makes the code logic easier to follow (and enables static checking for exhaustiveness... i just haven't turned this on in eslint yet)

```suggestion
    case 'gte':
    case 'gt':
    case 'lte':
    case 'lt':
      result = compareNumbers(field, condition, value)
      break
  default:
      throw new Error(
        `Conditional logic block contains an unknown operator: ${condition}`,
      )
```

Reply:
ok same as what i originally did, thanks!